packages/docs/PropTables/DropdownPropTable.tsx

```diff
@@ -132,8 +132,7 @@ const dropdownItemProps: Prop[] = [
   },
   {
     name: 'type',
-    types: "'text'",
-    defaultValue: "'text'",
+    types: 'text',
```

Comment:
I think we still want the `'` around this since it's referring to an enum value.
```suggestion
    types: "'text'",
```